includes/purecharity-wp-base.class.php

```diff
@@ -252,8 +252,8 @@ private function init_api() {
       self::$api_url = "https://staging.purecharity.com/api/";
     }
 
-    // self::$api_url = 'http://purecharity.ngrok.com/api/';
-    // self::$api_key = '5ccccb32e8e1e70e49d22ec3dae75544';
+    self::$api_url = 'http://purecharity.ngrok.com/api/';
+    self::$api_key = '5ccccb32e8e1e70e49d22ec3dae75544';
```

Comment:
@rafaeldalpra I wonder if we can instead do something similar to the dev addition I put on the demo instance.  By adding the development option we could allow the api_url and key to be assigned locally but used any time like this: 

```
  if ('production' == $mode) {
       self::$api_url = "https://purecharity.com/api/";  
  } elseif('sandbox' == $mode) {
       self::$api_url = "https://staging.purecharity.com/api/";
   } else {
     self::$api_url = $_ENV["API_URL"];
     }

```

Then update the options as well.  What do you think?